app/assets/stylesheets/pages/_roasts.scss

```diff
@@ -254,32 +255,39 @@ $range-handle-size-big: 67px;
         flex-direction: row;
         flex-wrap: wrap;
         justify-content: center;
-        margin-bottom: 5%;
+        margin-top: 20px;
+        margin-bottom: 35px;
         width: initial;
 
-        @include breakpoint(min-width 900px) {
-          justify-content: space-around;
+        @include breakpoint(min-width 800px) {
+          justify-content: center;
         }
 
-        @include breakpoint(min-width 900px) {
-          width: 50%;
+        @include breakpoint(min-width 700px) {
+          width: 100%;
         }
       }
 
       .brand__wrapper {
+        height: 200px;
+        margin-left: 1%;
+        margin-right: 1%;
+        margin-bottom: 5px;
         padding: 4px;
 
         img {
           border-radius: 10px;
         }
 
         .brand__text-wrapper {
-          position: relative;
           color: white;
           margin-bottom: 40%;
           margin-top: -60%;
```

Comment:
I've never liked using margin percentages with top and bottom. I sometimes run into issues using them this way. No issue if it works.